demos/weather_forecast/run_demo.sh

```diff
@@ -1,13 +1,36 @@
 #!/bin/bash
 set -e
 
+# Directory where the Docker Compose files are stored
+
+# Function to determine the docker-compose file based on the argument
+get_compose_file() {
+  case "$1" in
+    jaeger)
+      echo "docker-compose-jaeger.yaml"
```

Comment:
why are we echoing the value?

Reply:
That is the way I saw online for how people were returning from the function, i adjusted it so that it does not output to the user's terminal